src/any_llm/providers/anthropic/base.py

```
@@ -200,67 +261,56 @@ async def _amessages(
         if params.stream:
-            return self._stream_messages_async(**api_kwargs)
+            return self._stream_messages_async(use_beta=use_beta, **api_kwargs)
 
-        response: Message = await self.client.messages.create(**api_kwargs)
+        messages_resource = self.client.beta.messages if use_beta else self.client.messages
+        response: Message = await messages_resource.create(**api_kwargs)
         return self._convert_native_message_to_response(response)
 
-    async def _stream_messages_async(self, **kwargs: Any) -> AsyncIterator[MessageStreamEvent]:
+    async def _stream_messages_async(
+        self, *, use_beta: bool = False, **kwargs: Any
```

Ruff (0.16.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)